README.rst

```diff
@@ -87,6 +87,137 @@ There is built-in support for this option for Django >= 1.9. To use `sortkey`, s
 
 N.B.: there is no validation of this option, instead we let Redshift validate it for you. Be sure to refer to the `documentation <http://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_TABLE_examples.html>`_.
 
+Using distkey
+---------------------------------
+
+There is built-in support for this option for Django >= 1.11. To use `distkey`, define an index on the model
+meta with the custom index type `django_redshift_backend.base.DistKey` with `fields` naming a single field::
```

Comment:
I suggest `django_redshift_backend.models.DistKey` or `django_redshift_backend.DistKey`.

Because `django_redshift_backend.base` is internal name that reflects [django.db.backends.postgres.base](https://github.com/django/django/tree/master/django/db/backends/postgresql), we should not use `base` for users.

Reply:
Moved it to `django_redshift_backend.distkey.Distkey` if that's OK. `django_redshift_backend.models.DistKey` seemed wrong because this isn't a Django model. And `django_redshift_backend.DistKey` would require adding code in `__init__.py` which I prefer to keep empty. 

Does this make sense?

Reply:
Agreed. `django_redshift_backend.DistKey` is better.